cmd/meroxa/global/client.go

```diff
@@ -172,9 +173,31 @@ func NewClient() (meroxa.Client, error) {
 		}
 	}
 	options = append(options, meroxa.WithAccountUUID(Config.GetString(UserAccountUUID)))
+	options = append(options, meroxa.WithHeader("Meroxa-CLI", "1"))
```

Comment:
I updated the implementation plan, we can remove this: https://www.notion.so/Turbine-Data-Flow-Implementation-Plans-Phase-1-6d4780375ddc4ca99a5ce7a97200bb1e?d=b667c1685a2c47f081e6d6475474487a#1df09d39eac04b0b9bdaa06f2bf9c5dc